app/models/manageiq/providers/azure/cloud_manager.rb

```diff
@@ -46,4 +46,33 @@ def verify_credentials(_auth_type = nil, options = {})
 
     true
   end
+
+  # Operations
+
+  def vm_start(vm, _options = {})
+    group = vm.uid_ems.split('\\').first
+    vmm = ::Azure::Armrest::VirtualMachineService.new(vm.provider_object)
```

Comment:
Actually, this is really confusing.  provider_object should be returning the `::Azure::Armrest::VirtualMachineService` instance.  Then with_provider_object will yield it.